packages/wix-ui-tpa/src/components/DatePicker/DatePicker.visual.tsx

```diff
@@ -154,5 +154,15 @@ visualize('DatePicker', () => {
         today={new Date('2019/12/17')}
       />,
     );
+    snap(
+      'responsive rtl',
+      <div dir="rtl">
```

Comment:
<TPAComponentsConsumer>
```suggestion
      <div dir="rtl">
           <TPAComponentsProvider value={{ rtl: true }}>
```